packages/components/pie-cookie-banner/vite.config.js

```diff
@@ -2,18 +2,14 @@ import viteConfig from '@justeattakeaway/pie-components-config/vite.config';
 
 export default viteConfig({
     build: {
-        lib: {
-            entry: {
-                index: 'src/index.ts',
-                react: 'src/react.ts',
-            },
-            formats: ['es'],
-        },
         rollupOptions: {
             external: [
-                'react',
-                /^lit/,
-                /@justeattakeaway/
+                '@justeattakeaway/pie-button',
+                '@justeattakeaway/pie-divider',
+                '@justeattakeaway/pie-icon-button',
+                '@justeattakeaway/pie-link',
+                '@justeattakeaway/pie-modal',
+                '@justeattakeaway/pie-switch',
```

Comment:
I wonder if it would be reasonable to generate this lists by cross-referencing the component dependencies and the names of the modules stored in `packages/components`. It would save us from having to remind to check this.

Reply:
Yes, that's definitely a good idea, not sure how I'd implement it at the moment though. Once we start having lots of components and dependencies it would be good to make it more automatic for sure

Reply:
I've been looking into this and I'll raise a follow-up PR